### ISSUE TITLE
Docs are not displayed on new go.dev docs site due to license restrictions

### ISSUE THREAD

**Issue by goodspark** [DataDog/dd-trace-go#605]
https://pkg.go.dev/gopkg.in/DataDog/dd-trace-go.v1/ddtrace?tab=doc

https://pkg.go.dev/license-policy

**Comment by cgilmour** [comment]
Hi @goodspark thanks for reporting this.

This seems to be from changing to a dual license a little while ago.
The `LICENSE` file used to contain the contents of an OSI license, but now just contains a description of the dual-license.
The actual license files, `LICENSE.Apache` and `LICENSE.BSD3` aren't in the list of filenames that are checked for license detection.

A partial fix might be renaming `LICENSE.Apache` to `LICENSE-Apache`, so it matches a checked filename. There's no equivalent for BSD3 though.

**Comment by gbbr** [comment]
This should be fixed now. I think go.dev will pick it up once we do the next release. Closing for now. Thanks for taking the time to let us know.

**Comment by gbbr** [comment]
It looks like the fix didn't work. Re-opening. FWIW I've opened an issue here: https://github.com/google/licensecheck/issues/22. Let's see how that goes.

**Comment by gbbr** [comment]
Response from the mailing list is:
```
One solution: make LICENSE the concatenation of the two license files. Move the text about dual-licensing to the readme or somewhere else, or you can include it at the top of LICENSE (see below). You can leave LICENSE-APACHE and LICENSE-BSD3 as they are if you like.

What LICENSE can look like:
---------
## License

This work is dual-licensed under Apache 2.0 or BSD3, reproduced below.
You may select, at your option, one of the above-listed licenses.

`SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause`

                                 Apache License
                           Version 2.0, January 2004
[rest of Apache text, followed by BSD-3 text]
---------
We'll ignore the initial part (including the SPDX line) and recognize the two licenses.
```

**Comment by knusbaum** [comment]
This is fixed as of `v1.24.0`, and docs are now visible at pkg.go.dev 😄
